frontend/nexus/user.ts

```diff
@@ -85,6 +85,28 @@ schema.extendType({
       },
     })
 
+    t.field('updateUser', {
+      type: 'User',
+      args: {
+        userId: schema.intArg({ required: true }),
+        handle: schema.stringArg({ required: false }),
+        email: schema.stringArg({ required: false }),
+        name: schema.stringArg({ required: false }),
+        profileImage: schema.stringArg({ required: false }),
+      },
+      resolve: async (_parent, args, ctx: any) => {
+        const updates = { ...args }
+        delete updates.userId
+
+        return ctx.db.user.update({
+          data: updates,
+          where: {
+            id: args.userId,
```

Comment:
Cookie auth again, this will allow anyone to update anyone else's profile

Reply:
Done! 😄